lib/browsenator/remote/browserstack/base_browser.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'browserstack_credentials'
+
+class BaseBrowser
```

Comment:
Not sure this is a good case for inheritance as we just need to share one method but don't see the is-a relationship. Maybe we could include this method in the WindowSize module. It could also be a good idea to make it a class (value object).


```ruby
class WindowSize
  def initialize(width: , height:)
    @width = width || 1024
    @height = height || 768
  end

  def resolution
    "#{width}x#{height}"
  end
end
```

Maybe the module is just enough for this.

Reply:
If we still want to go for inheritance maybe we can apply it in a different way so that we can benefit more form it. Will look something like this:

```ruby
class BrowserstackBrowser
  def initialize
    ##put here the common attributes that repeat over the browsers and then call this with super
  end

  def open
    ## open code here
  end
end
```

Reply:
For now I just changed it back to use the window module, I think it makes sense to go for the second approach, maybe in another PR

Reply:
👍